homeassistant/components/xiaomi_miio/number.py

```diff
@@ -4,6 +4,7 @@
 from dataclasses import dataclass
 
 from homeassistant.components.number import NumberEntity, NumberEntityDescription
+from homeassistant.components.number.const import DOMAIN as PLATFORM_DOMAIN
```

Comment:
you can import this from `homeassistant.components.number` directly. also namei t `NUMBER_DOMAIN`